cornflow-server/cornflow/schemas/common.py

```diff
@@ -10,10 +10,19 @@ class QueryFilters(BaseQueryFilters):
 
 
 class BaseDataEndpointResponse(Schema):
-    id = fields.Str()
+    id = fields.Str(required=True)
     name = fields.Str()
     description = fields.Str()
     created_at = fields.DateTime()
-    user_id = fields.Int()
+    user_id = fields.Int(required=True)
     data_hash = fields.Str()
     schema = fields.Str(required=False)
+    updated_at = fields.DateTime()
+    deleted_at = fields.DateTime()
+    user_name = fields.Str(required=True)
+    last_name = fields.Str(required=True)
+    email = fields.Str(required=True)
+    role_id = fields.Int(required=True)
+    url_rule = fields.Str(required=True)
+    description = fields.Str(required=True)
+
```

Comment:
I think these changes are not needed. `BaseDataEndpointResponse` is used as the base of other schemas, so if you need to add fields, you should add them to the other schemas not these ones. For example you are making so that the instance, execution and case endpoints that have schemas that inherit from here should give back an `email` field as it is `required=True` so that does not make sense.